docs/version0.21.md

```diff
@@ -62,6 +63,9 @@ java.lang.NoClassDefFoundError: mypackage/Main (wrong name: Main)
 
 The version information for shared libraries on OSX has been updated from 0.0.0 to 1.0.0. If an application has linked against a shared library from a previous OpenJ9 release, it needs to be re-linked against the new release. Failure to re-link causes an error `Incompatible library version`, `requires version 0.0.0`.
 
+### New `-XX:[+|-]PrintFlagsFinal` option added
+This Hotspot option outputs the values of all configuration parameters. See [`-XX:[+|-]PrintFlagsFinal`](xxprintflagsfinal.md) for more details about this option.
```

Comment:
I think we should be clear that this option is incomplete at the moment, and falls far short of "outputs the values of all configuration parameters".

How about adding: "With this release, an initial implementation of this option is incomplete and outputs only a subset of parameters. Over time, we expect more options to be added to the output."

Reply:
..."options we have been told or found to be expected by various software projects and packages?"

sounds awkward, but maybe you can find a cleaner way to say that?